engine/src/main/resources/org/camunda/bpm/engine/db/upgrade/cockroachdb_engine_7.16_to_7.17.sql

```diff
@@ -16,4 +16,11 @@
 --
 
 insert into ACT_GE_SCHEMA_LOG
-values ('600', CURRENT_TIMESTAMP, '7.17.0');
\ No newline at end of file
+values ('600', CURRENT_TIMESTAMP, '7.17.0');
+
+-- https://jira.camunda.com/browse/CAM-14006 --
+ALTER TABLE ACT_RU_JOB 
+  ADD COLUMN LAST_FAILURE_LOG_ID_ varchar(64);
+  
+ALTER TABLE ACT_RU_EXT_TASK 
+  ADD COLUMN LAST_FAILURE_LOG_ID_ varchar(64);
```

Comment:
Due to https://github.com/cockroachdb/cockroach/pull/58182, we need to alter tables first and write to schema log afterward.

Reply:
Good catch, thank you.